cmake/PythonFixUpTclTk.py

```diff
@@ -63,9 +63,9 @@
 import stat
 
 
-def locate_tk_so(_python_dir: Path) -> Path:
-    print(f"Searching for _tkinter so in {_python_dir}")
-    sos = list(_python_dir.glob("lib-dynload/_tkinter*.so"))
+def locate_tk_so(python_dir: Path) -> Path:
+    print(f"Searching for _tkinter so in {python_dir}")
+    sos = list(python_dir.glob("lib-dynload/_tkinter*.so"))
```

Comment:
My renaming here was just to avoid variable shadowing from the outer scope.  But that's fine, it can go back to shadowing.